src/matlab/IKServer.m

```diff
@@ -26,7 +27,7 @@
     function obj = loadNominalData(obj,filename)
       if nargin < 1
         filename = [getenv('DRC_BASE'), ...
-          '/software/control/matlab/data/atlas_bdi_fp.mat'];
+          '/software/control/matlab/data/atlas_v3/atlas_bdi_fp.mat'];
```

Comment:
Yes.  We'd need to update a few locations that assume filename locations in software/control but I think this would be a good improvement, definitely.

And more generally, I'd like to move away from reading nominal pose data from .mat files, and as much as possible pass in the nominal pose as arguments.  We already do this in a lot of places, but there's probably still a few consumers of the .mat files.